### ISSUE TITLE
Getting error when running Docker Container

### ISSUE THREAD

**Issue by crippledfaith** [Mintplex-Labs/anything-llm#191]
fatal: not a git repository (or any of the parent directories): .git 

```
2023-08-16 00:06:45 [2023-08-15 18:06:45 +0000] [9] [INFO] Starting gunicorn 20.1.0
2023-08-16 00:06:45 [2023-08-15 18:06:45 +0000] [9] [INFO] Listening at: http://0.0.0.0:8888 (9)
2023-08-16 00:06:45 [2023-08-15 18:06:45 +0000] [9] [INFO] Using worker: sync
2023-08-16 00:06:45 [2023-08-15 18:06:45 +0000] [16] [INFO] Booting worker with pid: 16
2023-08-16 00:06:45 [2023-08-15 18:06:45 +0000] [17] [INFO] Booting worker with pid: 17
2023-08-16 00:06:45 [MIGRATIONS STUBBED] Please ping /migrate once server starts to run migrations
2023-08-16 00:06:45 [TELEMETRY ENABLED] Anonymous Telemetry enabled. Telemetry helps Mintplex Labs Inc improve AnythingLLM.
2023-08-16 00:06:45 [2023-08-15 18:06:45 +0000] [18] [INFO] Booting worker with pid: 18
2023-08-16 00:06:45 SELECT * FROM system_settings WHERE label = 'telemetry_id'
2023-08-16 00:06:45 fatal: not a git repository (or any of the parent directories): .git
2023-08-16 00:06:45 node:internal/errors:865
2023-08-16 00:06:45   const err = new Error(message);
2023-08-16 00:06:45               ^
2023-08-16 00:06:45 
2023-08-16 00:06:45 Error: Command failed: git rev-parse HEAD
2023-08-16 00:06:45 fatal: not a git repository (or any of the parent directories): .git
2023-08-16 00:06:45 
2023-08-16 00:06:45     at checkExecSyncError (node:child_process:885:11)
2023-08-16 00:06:45     at Object.execSync (node:child_process:957:15)
2023-08-16 00:06:45     at getGitVersion (/app/server/endpoints/utils.js:5:6)
2023-08-16 00:06:45     at setupTelemetry (/app/server/utils/database/index.js:103:13)
2023-08-16 00:06:45     at async Server.<anonymous> (/app/server/index.js:90:5) {
2023-08-16 00:06:45   status: 128,
2023-08-16 00:06:45   signal: null,
2023-08-16 00:06:45   output: [
2023-08-16 00:06:45     null,
2023-08-16 00:06:45     Buffer(0) [Uint8Array] [],
2023-08-16 00:06:45     Buffer(69) [Uint8Array] [
2023-08-16 00:06:45       102,  97, 116,  97, 108,  58,  32, 110, 111, 116,  32,
2023-08-16 00:06:45        97,  32, 103, 105, 116,  32, 114, 101, 112, 111, 115,
2023-08-16 00:06:45       105, 116, 111, 114, 121,  32,  40, 111, 114,  32,  97,
2023-08-16 00:06:45       110, 121,  32, 111, 102,  32, 116, 104, 101,  32, 112,
2023-08-16 00:06:45        97, 114, 101, 110, 116,  32, 100, 105, 114, 101,  99,
2023-08-16 00:06:45       116, 111, 114, 105, 101, 115,  41,  58,  32,  46, 103,
2023-08-16 00:06:45       105, 116,  10
2023-08-16 00:06:45     ]
2023-08-16 00:06:45   ],
2023-08-16 00:06:45   pid: 23,
2023-08-16 00:06:45   stdout: Buffer(0) [Uint8Array] [],
2023-08-16 00:06:45   stderr: Buffer(69) [Uint8Array] [
2023-08-16 00:06:45     102,  97, 116,  97, 108,  58,  32, 110, 111, 116,  32,
2023-08-16 00:06:45      97,  32, 103, 105, 116,  32, 114, 101, 112, 111, 115,
2023-08-16 00:06:45     105, 116, 111, 114, 121,  32,  40, 111, 114,  32,  97,
2023-08-16 00:06:45     110, 121,  32, 111, 102,  32, 116, 104, 101,  32, 112,
2023-08-16 00:06:45      97, 114, 101, 110, 116,  32, 100, 105, 114, 101,  99,
2023-08-16 00:06:45     116, 111, 114, 105, 101, 115,  41,  58,  32,  46, 103,
2023-08-16 00:06:45     105, 116,  10
2023-08-16 00:06:45   ]
2023-08-16 00:06:45 }
2023-08-16 00:06:45 
2023-08-16 00:06:45 Node.js v18.17.1
```

**Comment by timothycarambat** [comment]
This has been fixed as of `ec0567d9375f806bc1e1af0523443287e83e2642`! Please pull latest commit and re-build 👍

**Comment by crippledfaith** [comment]
Thanks Issue Resolved
